pkg/helpers/helpers.go

```diff
@@ -46,12 +47,34 @@ import (
 const (
 	defaultReplica = 3
 	singleReplica  = 1
+
+	ComponentHubKey   = "hub"
+	ComponentSpokeKey = "spoke"
+
+	ClusterClaim               featuregate.Feature = "ClusterClaim"
```

Comment:
@qiujian16 We need know registration's featrue gates.  So should we import registration project in registration-operator to use constants or variables in `open-cluster-management-io/registration/pkg/features/feature.go`?  Or, we write these constants or variables to open-cluster-management-io/api, then, we import  api project in registration and registration-operator?
And, registration project can not go get.

Reply:
I think we put those in api repo

Reply:
> 
@qiujian16 ok, I would move `open-cluster-management-io/registration/pkg/features/feature.go` to `open-cluster-management-io/api/operator/v1/feature.go`.  what do you think?